.github/workflows/regression_tests_docker.yml

```diff
@@ -0,0 +1,55 @@
+name: Run Regression Tests on Docker
+
+on:
+  # run every day at 5:15am
+  schedule:
+    - cron:  '15 5 * * *'
+
+concurrency:
+  group: ci-cpu-${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docker-regression:
+    strategy:
+      fail-fast: false
+      matrix:
+        hardware: [ubuntu-20.04, [self-hosted, regression-test-gpu]]
+    runs-on:
+      - ${{ matrix.hardware }}
+    steps:
+      - name: Clean up previous run
+        run: |
+          echo "Cleaning up previous run"
+          ls -la ./
+          sudo rm -rf ./* || true
```

Comment:
Nit: Why do we need the `|| true` part? Does this command fail in certain conditions and we want to ignore it?

Reply:
Yes, there are weird permission issues because of pycache geenrated by docker run. This logic worked for me. Will have to monitor the runs for the next few days